qiime/core/archiver.py

```diff
@@ -211,12 +210,13 @@ def save(self, filepath):
                     zf.write(abspath, arcname=archive_path)
 
     def _save_version(self, zf, root_dir):
+        version = '%s\n' % self._VERSION
         zf.writestr(os.path.join(root_dir, self._VERSION_FILENAME),
-                    '%s\n' % self._VERSION)
+                    version.encode('utf-8'))
```

Comment:
It does, but the weirdness is in that `str(bytes(b'a'))` -> `"b'a'"` (I guess to punish people who don't understand encoding?) and since the method is `writestr` I suspect it may be encoding already (I'm going to check this).